key.cc

```diff
@@ -104,7 +104,7 @@ TEST_F(ReadOnlySessionTest, WrapUnwrap) {
   SecretKey k2(session_, k2_attrs);
 
   // Use k2 to wrap k1.
-  CK_MECHANISM wrap_mechanism = {CKM_DES_ECB, NULL_PTR, 0};
+  CK_MECHANISM wrap_mechanism = {CKM_DES3_ECB, NULL_PTR, 0};
```

Comment:
This updates things a bit and helps with your use case, but it would be nice if this could be a bit more flexible for other scenarios.  [As previously suggested](https://github.com/google/pkcs11test/issues/34#issuecomment-770981722), it might be nice to:
 - add a new global or two to `globals.cc` to hold the default wrapping/keygen mechanism
 - use the global values instead of hardcoded constants here
 - add a command-line option that allows the global values to be overridden
    - (for bonus points, could allow the user to specify the value as a string and look up the corresponding mechanism info in `kCipherInfo`)

That way any future situations where a different key wrapping mechanism is needed could just be a case of testing with (say) `--wrap-with=AES-ECB`.

(If that's too much for you let me know and I'll try to have a go, but it might be a while.)

Reply:
I understand, I'll do it as soon as possible. Thanks a lot for the feedback.